manuscript.md

```diff
@@ -407,6 +409,7 @@ maps seem to agree with species interaction data. However, the disagreement
 between the IUCN and the GBIF databases is concerning and suggests that the IUCN
 maps might overestimate the lion's distribution.  
```

Comment:
underestimate, not overestimate

Reply:
Fixed on [f5a7f74](https://github.com/graciellehigino/ms_range_interactions/pull/98/commits/f5a7f74ebdcdc3c66f8a27826bbcf7e057737346)